### ISSUE TITLE
i18n visibility doesn't apply for footnotes

### ISSUE THREAD

**Issue by BlueGlassBlock** [YunYouJun/valaxy#423]
### Describe the bug

```md
## 脚注 {lang="zh-CN"}

## Footnote {lang="en"}

::: zh-CN

你可以使用 `[^1]` 或 `[^footnote]` 来添加脚注，例如：

这是一个脚注[^1-zh]。

这是一段脚注[^2-zh]。

[^1-zh]: 这是一个脚注。

[^2-zh]: 这是一段脚注。

  正确缩进的脚注段落会被自动附加。
:::

::: en

You can use `[^1]` or `[^footnote]` to add footnotes, for example:

This is a footnote[^1-en].

This is a paragraph of footnote[^2-en].

[^1-en]: This is a footnote.

[^2-en]: This is a paragraph of footnote.

  Footnote paragraphs with correct indentation will be automatically attached.
:::
```

produces

![image](https://github.com/user-attachments/assets/bb0b038f-cd0e-44d1-8b00-a539bcb91317)

### Reproduction

See above

### System Info

```Shell
Irrelevant
```

### Used Package Manager

pnpm

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by BlueGlassBlock** [comment]
IMO we should address this at valaxyjs/css-i18n instead

**Comment by YunYouJun** [comment]
I have no idea.
But it seems work:

```md
[^1]: <span class="inline-block" lang="zh-CN">中文</span><span class="inline-block" lang="en">English</span>
```

**Comment by BlueGlassBlock** [comment]
Anyway this issue is about https://github.com/valaxyjs/css-i18n/pull/1

**Comment by BlueGlassBlock** [comment]
Fixed in css-i18n v0.0.3
